routes/web.php

```diff
@@ -29,6 +29,7 @@
 
 //Admin
 Route::group(['prefix' => 'admin', 'as' => 'admin.'], function () {
+    Route::get('/attribute/show', [AttributesController::class, 'showAttribute'])->name('admin.attributes.show');
```

Comment:
Please see comments on PR # 17 regarding the redundant route naming :)

`->name('admin.attributes.show');` will look something like `admin.admin.attributes.show` when called